src/shaders/disk_andrews2009_frag.glsl

```diff
@@ -47,14 +47,11 @@ void main(void)
 {
     // Compute the radius and height of this fragment
     float radius = sqrt(ModelPosition.x * ModelPosition.x + ModelPosition.y * ModelPosition.y);
-    float height = abs(ModelPosition.z);
+    float height = atan(abs(ModelPosition.z),radius);
```

Comment:
On NVIDIA hardware there are only a handful (1-8) transidental function units for each group of 8-256 floating point units. Thus when transidental functions are used the execution on the GPU is serialized. This can degrade performance by as much as 87% so even though atan2 matches what they do in the paper, we should use the better performing option using the cartesian coordinates.